tests/package.json

```diff
@@ -29,6 +29,7 @@
     "@sku-fixtures/ssr-hello-world": "workspace:*",
     "@sku-fixtures/storybook-config": "workspace:*",
     "@sku-fixtures/styling": "workspace:*",
+    "@sku-fixtures/suspense": "workspace:*",
```

Comment:
Seems like maybe our `hoistWorkspacePackages: false` config wasn't working until now. This is necessary because a test file imports this package's sku config.